OTAcceleratorCore/OTAnnotation/Category/JSON.h

```diff
@@ -0,0 +1,15 @@
+//
+//  JSON.h
+//
+//  Copyright © 2016 Tokbox, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
Do we need a class like this?
It looks like JSON parsing is already in the iOS SDK.